docs/run/native-gates.mdx

```diff
@@ -0,0 +1,104 @@
+| [if_else](/api/qiskit/qiskit.circuit.IfElseOp) | control flow for classical feedforward |
+| [for_loop](/api/qiskit/qiskit.circuit.ForLoopOp) | control flow for classical feedforward |
+| [switch_case](/api/qiskit/qiskit.circuit.SwitchCaseOp) | control flow for classical feedforward |
+| [measure](/api/qiskit/qiskit.circuit.library.Measure) | |
+| [delay](/api/qiskit/qiskit.circuit.Delay) |  |
+
+### Eagle
+
+| Name |  Notes |
+| :--- | :--- |
+| [ECR](/api/qiskit/qiskit.circuit.library.ECRGate) | two-qubit gate |
+| [RZ](/api/qiskit/qiskit.circuit.library.RZGate) | single-qubit gate |
+| [SX](/api/qiskit/qiskit.circuit.library.SXGate)  |single-qubit gate |
+| [X](/api/qiskit/qiskit.circuit.library.XGate)  | single-qubit gate |
+| [ID](/api/qiskit/qiskit.circuit.library.IGate) | single-qubit gate wait cycle |
+| [reset](/api/qiskit/qiskit.circuit.library.Reset)  | single-qubit gate, non-unitary; not the same as the initialization done at the start of a circuit to prepare the all 0's state |
+| [if_else](/api/qiskit/qiskit.circuit.IfElseOp) | control flow for classical feedforward |
+| [for_loop](/api/qiskit/qiskit.circuit.ForLoopOp) | control flow for classical feedforward |
+| [switch_case](/api/qiskit/qiskit.circuit.SwitchCaseOp) | control flow for classical feedforward |
+| [measure](/api/qiskit/qiskit.circuit.library.Measure) | |
+| [delay](/api/qiskit/qiskit.circuit.Delay) |  |
+
+### Falcon
+
+| Name |  Notes |
+| :--- | :--- |
+| [CX](/api/qiskit/qiskit.circuit.library.CXGate)  | two-qubit gate |
+| [RZ](/api/qiskit/qiskit.circuit.library.RZGate) | single-qubit gate |
+| [ID](/api/qiskit/qiskit.circuit.library.IGate) | single-qubit gate wait cycle |
+| [reset](/api/qiskit/qiskit.circuit.library.Reset)  | single-qubit gate, non-unitary; not the same as the initialization done at the start of a circuit to prepare the all 0's state |
+| [if_else](/api/qiskit/qiskit.circuit.IfElseOp) | control flow for classical feedforward |
+| [for_loop](/api/qiskit/qiskit.circuit.ForLoopOp) | control flow for classical feedforward |
+| [switch_case](/api/qiskit/qiskit.circuit.SwitchCaseOp) | control flow for classical feedforward |
+| [measure](/api/qiskit/qiskit.circuit.library.Measure) | |
+| [delay](/api/qiskit/qiskit.circuit.Delay) |  |
+
+
+<Admonition type="tip" title="The init_qubits flag">
+
+By default, the `init_qubits` flag is set to True. If the `init_qubits` flag is set to False, you must insert [resets](/api/qiskit/qiskit.circuit.library.Reset) at the start of your circuit to start your qubits in the ground state. Otherwise, if your job consists of multiple circuits, the qubits will begin in their state from the previous shot. (If a job consists of multiple circuits, the shots will be executed in a "round-robin" fashion. That is, one shot will be collected from every circuit, and then the next shot will be collected from every circuit, and so on.)
```

Comment:
```suggestion
The `init_qubits` flag controls whether qubits are reset to the zero state at the start of each circuit. Its default value is True, indicating that the qubits should be reset. If False, the qubits will begin in the final state from the previous shot, and you must manually insert [resets](/api/qiskit/qiskit.circuit.library.Reset) if you want to reset them to the zero state. If a job consists of multiple circuits, then the shots are executed in a "round-robin" fashion. That is, each circuit will be executed in sequence to obtain one shot from each circuit. This process is then repeated until the requested number of shots has been obtained from all circuits.
```

Reply:
If we include this note, we have to mention what function the `init_qubits` flag is an argument of. A quick search of the source code suggests it is from [`assemble`](https://github.com/Qiskit/qiskit/blob/ec859e174ca2ac35f5c740895bf4040d5face47c/qiskit/compiler/assembler.py#L42). But is this function even mentioned elsewhere in the docs?

Reply:
Also, the Conditional Reset page (which has been removed) had this pph:

> **Note:** If `init_qubits` is set to false and you want qubits to start in the ground state, you must insert resets at the start of your circuit. Otherwise, qubits will begin with their state from the previous shot (after undergoing some thermalization process for the interval between shots as set by `rep_delay`). Note that the first shot of the first circuit may still be in the thermal ground state due to a longer delay existing during the loading of your job to hardware.

Reply:
So that's what I'm working with. Blake specifically mentioned the init_qubits flag, as long as it is supported by Sampler (which it is)... does it feel too much like it comes out of nowhere?

Reply:
> From reading the note, it's not clear to me in what sense the `init_qubits` flag is supported by Sampler. Is it `Sampler(init_qubits=False)`?

You specify it as an execution option: ExecutionOptions(shots=4000, init_qubits=True https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ExecutionOptions

Reply:
Ah ok, the note should say that.